src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -195,21 +195,24 @@ private IEnumerable<string> GetProjectsToMigrate(string projectArg)
             else if (projectArg.EndsWith(GlobalSettings.FileName, StringComparison.OrdinalIgnoreCase))
             {
                 projects =  GetProjectsFromGlobalJson(projectArg);
+                if (!projects.Any())
+                {
+                    throw new Exception("Unable to find any projects in global.json");
+                }
             }
             else if (Directory.Exists(projectArg))
             {
                 projects = Directory.EnumerateFiles(projectArg, Project.FileName, SearchOption.AllDirectories);
+                if (!projects.Any())
+                {
+                    throw new Exception($"No project.json file found in '{projectArg}'");
```

Comment:
Can you add tests for the two scenarios where you are throwing in dotnet-migrate.Tests